tests/test_atexitasync.py

```diff
@@ -16,6 +16,10 @@
         (signal.SIGINT, 5),
     ],
 )
+@pytest.mark.skipif(
+    sys.platform == "win32",
+    reason="Testing signals on Windows doesn't seem to be possible",
```

Comment:
What happens if you try to test this on Windows?

Reply:
The tests either fail or hang on Windows.

From what I have seen, on Windows, child processes of this test framework are unable to decode the signals being sent to them resulting in unstable behavior. eg: `test_asyncatexit` fails as opposed to `test_signals` which hangs.

This seems to be an artifact of the testing infrastructure, as our application that uses `jupyter-server-proxy` is able to accept and handle signals correctly on Windows.